### ISSUE TITLE
More distinctive period, line ending, and separation characters

### ISSUE THREAD

**Issue by nashpatty** [microsoft/cascadia-code#606]
Curious if there's any plans of making periods, commas, colons, semi-colons etc. more distinct.
Stylistically, these characters are easy to tell apart, but contextually, they're too thin, and easy to miss.
For example: 
cascadia mono
<img width="259" alt="Screen Shot 2021-12-06 at 1 31 50 PM" src="https://user-images.githubusercontent.com/28615881/144902096-e1a9e541-0ad8-4063-ac0b-aa7bbb900e00.png">
vs pragmata pro
<img width="238" alt="Screen Shot 2021-12-06 at 1 24 55 PM" src="https://user-images.githubusercontent.com/28615881/144901336-361889e7-f1af-4d8c-ae19-551fb1235f8c.png">

As you can see, the presence of the special characters is a lot clearer at a glance in the second picture, compared to the first.

Perhaps increasing the weight/size of these characters a little more would go a long way. Particularly for the period character. They're really well designed, but a tad bit too thin currently.

**Comment by aaronbell** [comment]
Tbh, I find the punctuation in pragmata pro overly dark. 

I’ll take another look at Cascadia’s but it’ll probably be a while before I can get to it.

**Comment by nashpatty** [comment]
I understand. Something between cascadia and pragmata would be the sweet spot. Perhaps a better example might have been fira or ibm plex for what I was thinking of.
Thanks for the reply, and great work on the font! :smile:

**Comment by pkazmier** [comment]
Yes, I agree with @nashpatty, some of the punctuation is just too light / small for a programming font. It’s a lovely font, but the period is just too small. It’s the only reason I don’t use the font.
